combat-trainer.lic

```diff
@@ -352,6 +355,12 @@ class LootProcess
     end
   end
 
+  def should_perform_ritual(game_state)
+    unless game_state.prepare_consume
+      (DRSkill.getxp('Thanatology') <= 30 || (@necro_train_first_aid && DRSkill.getxp('First Aid') <= 30)) || !@skin
+    end
```

Comment:
I considered making it default, but I decided to bench that discussion for now. I think 31 is fine if we drop the @skin check and use the simplified logic which will keep doing rituals while either skill needs training.